### PR TITLE
Add 0.61 blog post

### PR DESCRIPTION
**Pull request by rickhanlonii**
## Overview

Adds the blog post for 0.61

## Screen

localhost_3000_react-native_blog_2019_09_18_version-0 61 (6)

### CODE REVIEW

File: website/blog/2019-09-18-version-0.61.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+- Fast Refresh always remounts class components without preserving state. This ensures it works reliably.
+- We all make mistakes in the code! Fast Refresh automatically retries rendering after you save a file. You don't need to reload the app manually after fixing a syntax or a runtime error.
+- Adding a `console.log` or a `debugger` statement during edits is a neat debugging technique.
+
+Please report any issues with Fast Refresh on GitHub, and we’ll look into them.
+
+## Other Improvements
+
+- **Fixed use_frameworks! CocoaPods support.** In 0.60 we made some updates to integrate CocoaPods by default. Unfortunately, this broke builds using [use_frameworks!](https://guides.cocoapods.org/syntax/podfile.html#use_frameworks_bang). This is [fixed in 0.61](https://github.com/facebook/react-native/pull/25619), making it easier to integrate React Native into your iOS projects that require building with dynamic frameworks.
+- **Add useWindowDimensions Hook. **This new Hook automatically provides and subscribes to dimension updates, and can be used instead of the Dimensions API in most cases. See [x] for more information.
```
Comment (rickhanlonii) [inline website/blog/2019-09-18-version-0.61.md]: Waiting for a link to these docs

Comment (gaearon) [inline website/blog/2019-09-18-version-0.61.md]: Borked markdown header

Comment (TheSavior) [reply website/blog/2019-09-18-version-0.61.md]: Typo here keeping it from being bold. Shouldn’t have a space before the closing asterisks

Comment (rickhanlonii) [reply website/blog/2019-09-18-version-0.61.md]: I just removed the mention of docs

**Comment by brentvatne** [comment]
might be worth mentioning that this is the first release without haste:
- https://github.com/facebook/react-native/issues/18598
- https://twitter.com/JI/status/1136369775083319296

File: website/blog/2019-09-18-version-0.61.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+## Breaking Changes
+
+- **Remove React .xcodeproj.** In 0.60, we introduced auto-linking support via CocoaPods. We have also integrated CocoaPods into the e2e tests runs, so that from now on, we have a unified standard way of integrating RN into iOS apps. This effectively deprecates the React .xcodeproj support, and the file has been removed starting 0.61. Note: if you use 0.60 auto-linking already, you shouldn't be affected.
+
+## Thanks
+
+Thanks to all of the contributors that helped make 0.61 possible!
+
+To see all the updates, take a look at the 0.61 changelog.
```
Comment (kelset) [inline website/blog/2019-09-18-version-0.61.md]:
```suggestion
To see all the updates, take a look at the [0.61 changelog](https://github.com/react-native-community/releases/blob/master/CHANGELOG.md).
```

Comment (rickhanlonii) [reply website/blog/2019-09-18-version-0.61.md]: Thanks! I was waiting to add the link for when the PR is merged and we can link directly to the 0.61 header in the changelog, but I'll merge this and update it later 👍

**Comment by rickhanlonii** [comment]
Fast Refresh guide here: https://github.com/facebook/react-native-website/pull/1282

**Pull request merged**